dbms/src/Storages/Transaction/KVStore.h

```diff
@@ -77,6 +81,30 @@ class RegionPersister;
 struct CheckpointInfo;
 using CheckpointInfoPtr = std::shared_ptr<CheckpointInfo>;
 
+enum class PersistRegionReason
+{
+    Debug,
+    UselessAdminCommand, // Does not include passive CompactLog
+    AdminCommand,
+    Flush, // passive CompactLog
+    ProactiveFlush,
+    ApplySnapshotPrevRegion,
+    ApplySnapshotCurRegion,
+    IngestSst
+};
+
+constexpr const char * PersistRegionReasonMap[magic_enum::enum_count<PersistRegionReason>()] = {
+    "debug",
+    "admin cmd useless",
+    "admin raft cmd",
+    "tryFlushRegionData",
+    "ProactiveFlush",
```

Comment:
```suggestion
    "proactive flush",
```

Reply:
And also rename "tryFlushRegionData" to "passive flush"?

Reply:
I suggest it be renamed after we introduced the proactive flush PR, since the "flush" is the only "flush" here

Reply:
I will delete the "ProactiveFlush" reason, since there is no `ProactiveFlush` in this PR.